terraform/environments/bootstrap/single-sign-on/policies.tf

```diff
@@ -344,14 +335,9 @@ data "aws_iam_policy_document" "data_engineering_additional" {
       "glue:UpdateDatabase",
       "glue:UpdatePartition",
       "glue:UpdateTable",
-      "glue:CreateUserDefinedFunction",
-      "glue:DeleteUserDefinedFunction",
-      "glue:UpdateUserDefinedFunction",
-      "glue:CreateJob",
-      "glue:DeleteJob",
+      "glue:*DefinedFunction",
+      "glue:*Job",
       "glue:Get*",
-      "glue:*JobRun",
```

Comment:
are we losing this permission? 🤔

Reply:
I think this might be causing an issue for me. Today I've started to get a `not authorized to perform: glue:StartJobRun on resource` error when I try to run Glue jobs using the modernisation-platform-data-eng SSO role